package.json

```diff
@@ -33,12 +33,13 @@
     "gpsoauthnode": "0.0.5",
     "long": "^3.2.0",
     "node-pogo-protos": "^2.4.2",
-    "node-pogo-signature": "^4.0.0",
+    "node-pogo-signature": "pogosandbox/node-pogo-signature",
     "request": "^2.79.0",
     "s2-geometry": "^1.2.9"
   },
   "devDependencies": {
-    "eslint": "^3.12.2",
+    "eslint": "^3.13.1",
+    "eslint-config-airbnb-base": "^11.0.1",
```

Comment:
I don't know... Did you remove airbnb base at some point?
Tell me if it should be here or not.